jmclient/jmclient/storage.py

```diff
@@ -282,31 +309,18 @@ def _hash_password(cls, password, salt=None):
     def _create_lock(self):
         if self.read_only:
             return
-        (path_head, path_tail) = os.path.split(self.path)
-        lock_filename = os.path.join(path_head, '.' + path_tail + '.lock')
-        self._lock_file = lock_filename
-        if os.path.exists(self._lock_file):
-            with open(self._lock_file, 'r') as f:
-                try:
-                    locked_by_pid = int(f.read())
-                except ValueError:
-                    locked_by_pid = None
-            self._lock_file = None
-            raise RetryableStorageError(
-                               "File is currently in use (locked by pid {}). "
-                               "If this is a leftover from a crashed instance "
-                               "you need to remove the lock file `{}` manually." .
-                               format(locked_by_pid, lock_filename))
+
         #FIXME: in python >=3.3 use mode x
```

Comment:
This is kinda out of scope of this PR, but it is possible probably to rewrite this nicer using `open` mode `x` instead of `w` as we already require Python >3.3. See https://docs.python.org/3/library/functions.html#open.